dotnet/Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM gitpod/workspace-full:latest
 
 USER root
-RUN wget -q https://packages.microsoft.com/config/ubuntu/19.04/packages-microsoft-prod.deb && \
+
+RUN wget -q "https://packages.microsoft.com/config/ubuntu/19.04/packages-microsoft-prod.deb" && \
     dpkg -i packages-microsoft-prod.deb && \
     rm -rf packages-microsoft-prod.deb && \
     add-apt-repository universe && \
     apt-get update && \
-    echo resolvconf resolvconf/linkify-resolvconf boolean false | debconf-set-selections && \
-    DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get -y -o APT::Install-Suggests="true" install \
+    DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get --no-install-recommends -y -o APT::Install-Suggests="true" install \
```

Comment:
In my experience, `--no-install-recommends` will fail to install packages that are expected by downstream developers and programs.

For example, this was the reason that https://github.com/MovingBlocks/Terasology mysteriously refused to work in Gitpod. It only started working again when we removed `--no-install-recommends`. So, I don't like this option (sure, it saves a few bytes in the image, but is that really worth it if developers then spend hours/days struggling to get their applications working?)

Reply:
Done!